### DISCUSSION TITLE
Brackets 2.0.0 November Pre-release

### DISCUSSION THREAD

**Discussion by abose** [brackets-cont/brackets#65] [Announcements]
**_Brackets 2.0 first prerelease is here!_**

This prerelease is for windows only (Mac Prerelease coming soon). This pre-release includes
1. New installer experience for windows.
2. New brackets extension store maintained by the community.
3. Performance improvement for the files panel.
4. Removed PHP extension from brackets. Will be distributed through the extension store.

<hr /><em>This discussion was created from the release <a href='https://github.com/brackets-cont/brackets/releases/tag/v2.0.0-prerelease'>Brackets 2.0.0 November Pre-release </a>.</em>

**Comment by ScanuNicco** [comment]
Just installed it and overall it seems pretty good, but there are a few things I noticed:
- The installer gives an error if an older version is installed. This is not super urgent, but it would be nice if it could automatically uninstall the old version in the future.
- The grey text near the progress bar on the installer is hard to read
- The blue Brackets Health Report popup that opens on first launch still says data is sent to Adobe. 
- Checking for updates still fails (#44)

**Reply by abose** [reply]
Thanks for the detailed checkup @ScanuNicco  :) Let me update the issues for each.

**Comment by TheKodeToad** [comment]
Are Linux builds planned to be released in the future?

**Reply by abose** [reply]
Release 2.0 is only for Win/Mac. But we are working on a larger platform upgrade with the project here: https://github.com/brackets-cont/brackets/projects/3

Electron Port will take a couple of months to complete, and we will have Linux builds soon after.

**Reply by TheKodeToad** [reply]
Thank you for the quick reply. When it releases on Linux, I plan to start using it (if it's usable yet, if not, I can help with bug reporting), as it is what I used before Atom, and had many useful features.

**Reply by charlypa** [reply]
@TheKodeToad 
Thanks for your interest.  Migrating brackets to Electron is a top priority for better stability and maintenance. We will take up electron migration once this release is over.

Could you share which distirbution of Linux you are using?

**Reply by PunyTRexArms** [reply]
I've been using Brackets in Ubuntu since its sprint days. I'm absolutely in love with Brackets' live preview in Firefox, as I feel it is superior to that found in Visual Studio Code. So glad this project is still alive--hats off to the developers.

**Reply by charlypa** [reply]
@PunyTRexArms  Thanks for feedback

**Comment by arlindocodinha** [comment]
I have installed brackets 1.14.2 and 2.0.0 and both have the same issue after 1 minute start html page turns into a gray page and freezies. 

![image](https://user-images.githubusercontent.com/27304872/142789172-905dfaef-7f05-4d06-95dd-025b1f4128e8.png)

**Reply by arlindocodinha** [reply]
this is an old issue with 1.14 for me

**Reply by charlypa** [reply]
@arlindocodinha 
Apology for the inconvenience.
Could you please give the following information to debug the issue?


1. Have you recently updated OS?
2. Could you share the screenshot of the issue?

**Reply by arlindocodinha** [reply]
yes is the last version macOS 12.01
![image](https://user-images.githubusercontent.com/27304872/143143408-49558b55-170f-430b-9372-a94d31b9a820.png)

**Reply by arlindocodinha** [reply]
Thank you for everything

**Reply by charlypa** [reply]
@arlindocodinha 
Thanks for the update we will look in to latest version of mac

**Reply by charlypa** [reply]
@arlindocodinha 
Can you try to use the latest version of brackets from https://brackets.io/ and check if this issue still exists?

**Comment by arlindocodinha** [comment]
Hi yes I tested brackets 2.0.1 on other Mac with the latest Os version and
I Have the same issue grey screen.

**Reply by charlypa** [reply]
@arlindocodinha 
Apologies for that. Could you share your screen resolution?

**Comment by arlindocodinha** [comment]
It is a 5k iMac.

**Reply by charlypa** [reply]
@arlindocodinha 
We don't have this hardware with us.  Could you please try to reduce the screen resolution and see if it works?

**Reply by abose** [reply]
The screenshots look like the CEF instance failed to startup, which usually happens when brackets have crashed before by some means. Does this issue happen after a restart as well?

**Reply by charlypa** [reply]
@arlindocodinha 
Thanks for reporting the issue.
We have created an issue to track the same. Apologies for the trouble caused. 
https://github.com/brackets-cont/brackets/issues/94

**Comment by arlindocodinha** [comment]
This star ti happen with version 1.12 or 1.13

**Comment by arlindocodinha** [comment]
Thank you

**Comment by arlindocodinha** [comment]
No, I have scaled down the resolution of the screen and it is same

**Comment by arlindocodinha** [comment]
Just freezes.

**Comment by StephenESC** [comment]
I’m having the same issue on a 5K iMac Pro. I also just installed in on a 2017 MacBook Air and the same thing happened—grey window with the three buttons at the top bit nothing else. I also can’t close the application without force-quitting it.

I also confirmed it wasn’t a plugin conflict as I removed all my plugins and tried again without success.

**Reply by charlypa** [reply]
@StephenESC 
Thanks for reporting the issue. We have created an issue to track the same 
https://github.com/brackets-cont/brackets/issues/94

**Reply by charlypa** [reply]
@StephenESC 
Could you please share the `System Report` to debug the issue in detail?

**Reply by StephenESC** [reply]
Sorry I missed this earlier. Everything is working now!
Thanks! 😊

**Comment by abose** [comment]
@StephenESC @arlindocodinha 
We have pushed a new release with the fix for the issue. 
Please download the new build from brackets.io and see if the issue is fixed. Thanks :)

**Reply by StephenESC** [reply]
Everything is working now!
Thanks! 😊

**Comment by arlindocodinha** [comment]
Hi Arun,
to me still the same grey window. Is it the build 2.0.1?
Thank you

**Comment by arlindocodinha** [comment]
I uninstalled Brackets previous versions 2.0.0_rc worked for a few seconds uninstalled again, installed version 2.0.1 latest opened as soon as I tried to open a file back to the grey window and freezes.

**Reply by arlindocodinha** [reply]
This is a very old issue

**Reply by charlypa** [reply]
@arlindocodinha 
There was some issue with the Apple signature; we fixed it. Could you freshly download  2.0.1 again from brackets.io and try it again
